ContributeDocumentation.md

```diff
@@ -0,0 +1,88 @@
+
+# How to Preview Your Change
+
+In order to preview your documentation change, it is necessary to build the documentation and view it in a web browser. 
+The next step depends on whether you already have an existing ISIS build, or if you need to create 
+a new ISIS build. 
+
+## Building ISIS Documentation
+
+### Creating a New ISIS build: 
+
+Building ISIS requires that the anaconda environment you created in [Anaconda and ISIS3 Dependencies](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#anaconda-and-isis3-dependencies) be activated.
+
+Activate your anaconda environment with:
+
+`conda activate <environment-name>`
+
+* Navigate to the top-level directory of your ISIS3 clone (i.e `cd ISIS3`)
+
+* Create a `build` and an `install` directory at this level:
+  * `mkdir build install`
+
+* Navigate to the build directory: `cd build` 
+* Run the following command to configure your build:
+  * `cmake -Disis3Data=<path-to-isis3-data> -Disis3TestData=<path-to-isis3-test-data> -DJP2KFLAG=OFF -DCMAKE_BUILD_TYPE=RELEASE -GNinja ../isis`
+
+* Set your ISISROOT to `/the/path/to/your/build`:
+  * `export ISISROOT=$(pwd)`
+
+* And finally, build the documentation by running:
+
+```
+ninja docs -j7
+```
+
+It is not necessary to build the rest of ISIS to build and preview documentation-only changes. 
+
+### If You Already Have an Existing ISIS Build: 
+
+To build and view your documentation changes, first `cd` to your `$ISISROOT` directory in the terminal and run the following: 
+
+```
+ninja docs -j7
+```
+
+If you configured for `make`, instead of `ninja`, run:
+
+```
+make docs -j7
+```
+
+The built documentation will be available in the `docs` directory below the current build directory. 
+(If the release type was set to install during the build process, the built docs will also be copied over into `install/docs`, where `install` refers to the installation directory specified during the build.)
+
+
+## Previewing the Built Documentation
+To view your documentation, open the `index.html` file immediately under the docs directory (`$ISISROOT/docs/index.html`) using a web browser and use the web browser to navigate to the modified file. It is possible to do this by:
+
+* running either of the following in a terminal: `google-chrome index.html` or `firefox index.html`
+* copy-and-pasting the full file path to `index.html` (for example: `/home/username/dev/ISIS3/isis-build/docs/index.html`) into the address bar of a web browser.
+
+
+If you see changes that need to be made to the documentation at this point, you can make the changes in your documentation XML files, re-run your documentation build command, and then just refresh the web-page in your browser to view your updated changes. 
```

Comment:
With only changes to one app.xml, this should be quick, but is it possible to specify the exact target "ninja spiceinit.html"